packages/components/src/color-palette/index.js

```diff
@@ -21,10 +21,14 @@ export default function ColorPalette( {
 	className,
 	colors,
 	disableCustomColors = false,
-	onChange,
+	colorPickerMode = 'hex',
+	onColorChange,
```

Comment:
It looks like a breaking change, see: https://developer.wordpress.org/block-editor/how-to-guides/backward-compatibility/#how-to-preserve-backward-compatibility-for-a-react-component

Reply:
Thanks for the guidance! 😸 
I'll try to preserve the naming/method as it's now and adding the new one.